pkg/stub/handler.go

```diff
@@ -422,6 +457,17 @@ func (h *Handler) processFiles(dir string, files []os.FileInfo, opcfg *v1alpha1.
 		}
 
 		if strings.HasSuffix(dir, "templates") {
+			// allow setting of product dir as found in tmp/build/assests/operator as a way to skip as set of templates
+			// i.e. .../cakephp/templates/...
+			// setting "cakephp" will skip all the templates under cakephp/templates
```

Comment:
same concern here.  What was the motivation for adding this?

Reply:
explained above ... will remove